### PR TITLE
analytics improvements

### PR DESCRIPTION
**Pull request by mohsenParvar**

### CODE REVIEW

**Comment by ghost** [comment]
Can you check with @saito-sv why the live deployment failed for this PR? @mohsenParvar

File: yarn.lock [hunk 1]
```diff
@@ -4,15 +4,15 @@
 
 "@ampproject/toolbox-core@2.7.4", "@ampproject/toolbox-core@^2.7.1-alpha.0":
   version "2.7.4"
-  resolved "https://registry.yarnpkg.com/@ampproject/toolbox-core/-/toolbox-core-2.7.4.tgz#8355136f16301458ce942acf6c55952c9a415627"
+  resolved "https://registry.npmjs.org/@ampproject/toolbox-core/-/toolbox-core-2.7.4.tgz"
```
Comment (saito-sv) [inline yarn.lock]: this could be causing the build to failed.

**Review by saito-sv — CHANGES_REQUESTED** [review]
yarn.lock has too many changes unrelated to your task.

**Review by saito-sv — APPROVED** [review]
LGTM

**Pull request merged**